src/vm/builtins/builtin_runner/output.zig

```diff
@@ -24,14 +92,271 @@ pub const OutputBuiltinRunner = struct {
     /// # Returns
     ///
     /// A new `OutputBuiltinRunner` instance.
-    pub fn init(included: bool) Self {
+    pub fn from(included: bool, allocator: Allocator) Self {
         return .{
             .base = 0,
             .stop_ptr = null,
             .included = included,
+            .pages = AutoHashMap(usize, PublicMemoryPage).init(allocator),
         };
     }
 
+    /// Initializes segments for the OutputBuiltinRunner instance using the provided MemorySegmentManager.
+    ///
+    /// This function sets the base address for the OutputBuiltinRunner instance by adding a segment through the MemorySegmentManager.
+    ///
+    /// # Arguments
+    ///
+    /// - `self`: A pointer to the OutputBuiltinRunner instance.
+    /// - `segments`: A pointer to the MemorySegmentManager managing memory segments.
+    ///
+    /// # Returns
+    ///
+    /// An error if the addition of the segment fails, otherwise sets the base address successfully.
+    pub fn initializeSegments(self: *Self, segments: *MemorySegmentManager) !void {
```

Comment:
what do you think about mapping uses of the word `initialize` in implementations to `init` in zig? I searched through tigerbeetle and the zig code base and that seems to be the convention. 

so here we would do  `initializeSegments` -> `initSegements`

Reply:
Yep agreed, I just hadn't made the change afterwards because it was the same in the builtin keccak but I just changed everything like that it's done.